dbt_sugar/core/task/doc.py

```diff
@@ -191,17 +196,23 @@ def document_columns(self, columns: Dict[str, str]) -> None:
         columns_names = list(columns.keys())
         for i in range(0, len(columns_names), NUMBER_COLUMNS_TO_PRINT_PER_ITERACTION):
```

Comment:
you know that you can use `ennumerate()` for this kind of thing right?

Reply:
but is not the same, im getting the element by groups with this:
so if NUMBER_COLUMNS_TO_PRINT_PER_ITERACTION = 10
and we have 50 columns
this for will give me 0-9 in the first interaction, then 10-19...

Reply:
and I am not interested in the element, only in the index to pass it to the function.

Reply:
Ah yeah because of pagination!